socialhome/content/models.py

```diff
@@ -118,6 +118,8 @@ def save(self, author=None, *args, **kwargs):
             # New with author, set a GUID and author
             self.guid = uuid4()
             self.author = author
+            if self.pinned:
+                self.order = Content.objects.filter(pinned=True).count() + 1
```

Comment:
No, that would be the count of objects. You need the largest value. Check https://docs.djangoproject.com/en/1.10/ref/models/querysets/#max